tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -34,6 +34,33 @@ __standard_vector_size__: int
 STANDARD: ExplainType
 ANALYZE: ExplainType
 
+sqlnull: DuckDBPyType
```

Comment:
Is there a particular reason we want to expose them here? It feels like we're polluting the module namespace, maybe they should be in a submodule or just left on the enum

Reply:
I'd prefer having an easy way to use them, but you don't often need to interact with them, definitely not in a standard workflow. So maybe `duckdb.typing` could be a good place for it

Reply:
I'm looking at this now, and I'm not sure if it's a good idea to move these to a submodule, because we can't move the creation methods.
Because of extension/user defined types, we might need the ClientContext to parse those types.
I guess we could move the methods to the submodule as well and then let the methods just take an optional connection method...
Which wouldn't be horrible actually